src/main/java/com/mailchimp/domain/Lists.java

```diff
@@ -1,23 +1,22 @@
 package com.mailchimp.domain;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
-import java.util.List;
 import lombok.Data;
 
 @Data
-public class SubscriberLists {
+public class Lists {
```

Comment:
SubscriberLists seems a bit more clearer. The word `Lists` can be both a verb and noun. Also `Lists` by itself gives no indication of what is in the List.